azavea/temperate
# Refactor for add city request email

## Overview

Send add city request to our support email as HTML email; refactor user email sending to support HTML email for other destination addresses than the user.

Put user email in message, and also set it in the reply-to address on email sent.

### Demo

Log message about the email (puts the city name in the subject):
![add_city_email_sent](https://user-images.githubusercontent.com/960264/37630207-a8a702fc-2bb9-11e8-9912-4113dad68f52.png)

HTML email:
![request_add_html_email](https://user-images.githubusercontent.com/960264/37630213-ac429ff2-2bb9-11e8-8e86-a16f3749b9a3.png)


## Testing Instructions

 * Submit form to request to add a city at http://localhost:4210/create-organization
 * Check server logs for information about message
 * Check HTML email sent via link; text substitution should have worked



Closes #904.
